src/ext/property-panel/data.js

```diff
@@ -208,12 +208,12 @@ const getColumnResize = {
     type: "number",
     expression: "optional",
     defaultValue: 20,
-    show: (data) => data.qDef.columnWidth?.type === ColumnWidthTypes.PERCENTAGE,
+    show: (data) => data.qDef.columnWidth?.type === ColumnWidthType.Percentage,
     change(data) {
       data.qDef.columnWidth.percentage =
         data.qDef.columnWidth.percentage === undefined
           ? data.qDef.columnWidth.percentage
-          : Math.max(1, Math.min(MAX_COLUMN_PERCENTAGE_WIDTH, data.qDef.columnWidth.percentage));
+          : Math.max(1, Math.min(ColumnWidthValues.PixelsMax, data.qDef.columnWidth.percentage));
```

Comment:
`ColumnWidthValues.PixelsMax`?

Reply:
that is indeed wrong, I think I need to test this a bit more in the client setting values using the pp.